live/liveview.go

```diff
@@ -244,8 +349,9 @@ func (s *socket) dispatch(ctx context.Context, msg *phx.Msg) ([]byte, error) {
 			// headers or the like(?)
 			r := s.req.Clone(s.req.Context()) // todo: background context?
 			r.URL = url
-			s.view, err = s.config.viewForRequest(r, nil)
-			if err != nil {
+			var code int
+			s.view, code = s.config.viewForRequest(nil, r, nil)
+			if code%100 == 5 {
```

Comment:
seems like we only care if code is 5XX.  implies returning error from `viewForRequest` as suggested above could be better.  also error msg on next line seems incorrect?  or is at least confusing considering "not found" is 404 in HTTP status land.

Reply:
Yeah, I should reword 355.